orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/handler/PipelineTemplateErrorHandler.kt

```diff
@@ -31,7 +31,6 @@ class PipelineTemplateErrorHandler : Handler {
 
     if (context.getErrors().hasErrors(context.getRequest().plan)) {
       context.getProcessedOutput().putAll(context.getErrors().toResponse())
-      context.getProcessedOutput().put("mptRequestId", context.getRequestId())
```

Comment:
Should be removed from the `context` class as well.

Reply:
Done